src/models.py

```diff
@@ -16,24 +24,69 @@
 
 logger = logging.getLogger(__name__)
 
+class MLP(nn.Module):
+    def __init__(self, num_hidden):
+        super().__init__()
+        self.num_hidden = num_hidden
+
+        self.lin1 = nn.Linear(num_hidden, num_hidden)
```

Comment:
```suggestion
        self.lin1 = nn.Linear(num_hidden, num_hidden)
        self.relu1 = nn.ReLU(inplace=True)
        self.lin2 = nn.Linear(num_hidden, num_hidden)
        self.relu2 = nn.ReLU(inplace=True)
```
Replace one layer MLP with two layer MLP.